include/miopen/miopen.h

```diff
@@ -1730,6 +1730,14 @@ miopenFindConvolutionForwardAlgorithm(miopenHandle_t handle,
  * The scaling parameter alpha (float) and shift parameter beta (float) are only supported for
  * alpha = 1 and beta = 0.
```

Comment:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/8be2998d7a1210b7bc82d778f7df0d92aa65b57d/include/miopen/miopen.h#L667

Here mention that is allows creation of unpacked tensor.

Maybe this is a good place to give a definition of unpacked tensor.